packages/jsapi-components/src/HookTestUtils.tsx

```diff
@@ -1,11 +1,18 @@
-import React, { ReactNode } from 'react';
+import React, { forwardRef, ReactNode } from 'react';
 import { ApiContext } from '@deephaven/jsapi-bootstrap';
 import type { dh as DhType } from '@deephaven/jsapi-types';
 
-export function makeApiContextWrapper(dh: DhType) {
-  return function ApiContextWrapper({ children }: { children?: ReactNode }) {
+export function makeApiContextWrapper<TProps, TRef = unknown>(dh: DhType) {
+  return forwardRef<TRef, TProps>(function ApiContextWrapper(
+    {
+      children,
+    }: {
+      children?: ReactNode;
+    },
+    _ref
```

Comment:
I wonder what's the root cause of the issue? Why does `renderHook` require the ref? `_ref` argument isn't even passed anywhere, could this be a bug in `renderHook` itself?

Reply:
I suspect it uses it internally for something but not sure

Reply:
nvm, I was wrong. Real cause of the original issue is that `ref` needs to be excluded from the auto proxy